subsys/bootloader/provision/provision_flash.c

```diff
@@ -37,34 +48,57 @@ u32_t num_public_keys_read(void)
 	return p_provision_data->num_public_keys;
 }
 
+static const u32_t *get_key(u32_t key_idx)
+{
+	return &(p_provision_data->key_data[key_idx *
```

Comment:
This is a bit hard to read. Could we make it easy for people like me? :D 

```
size_t index_of_key_data = key_idx * (CONFIG_SB_PUBLIC_KEY_HASH_LEN / 4 + 1);
return &(p_provision_data->key_data[index_of_key_data]);
```

And why do we have `CONFIG_SB_PUBLIC_KEY_HASH_LEN / 4 + 1` ?

Reply:
Added defines and comments.